kernel/float64.mli

```diff
@@ -29,13 +29,17 @@ val to_hex_string : t -> string
 (** Print a float as a decimal value. The printing is not exact (the
  * real value printed is not always the given floating-point value),
  * however printing is precise enough that forall float [f],
- * [of_string (to_decimal_string f) = f].  *)
+ * [of_string (to_decimal_string f) = f]. *)
 val to_string : t -> string
 
 val compile : t -> string
 
 val of_float : float -> t
 
+(** All NaNs are normalized to [Pervasives.nan].
```

Comment:
Shouldn't this ocamldoc string be rephrased? (as `Pervasives` → OCaml < 4.07)
```suggestion
(** All NaNs are normalized to [Stdlib.nan].
```

Reply:
We're still technically supporting OCaml up to 4.05.0, so it depends on the context. I agree that this is more future-proof though.

Reply:
Done